docusaurus/docs/cloud/projects/overview.md

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
@gpene I'm wondering where should we add info about the Project suspension ? Maybe we can add it somewhere on this page, or create a separate "Project suspension" page under the Projects section, wdyt? 

Shortly about it: a project can be _suspended_ for different reasons, such as:
- payment issues (multiple different issues here actually)
- reaching the usage limits during the trial
- abuse

When a project is suspended, it means that until it is _resumed_: 
- your app is not reachable anymore, 
- you can't trigger a new deploy, neither manually nor by push
- you can't access logs

The only way to resume the project is to contact our Support team. 

Another important thing to document: **project will be deleted after 10 days of being suspended**. Which means user need to contact Support & fix the suspension reason before that time if they would like to avoid project deletion.

Reply:
Thanks @gpene, I missed that. I think it's perfectly fine to have it there 👍